.travis.yml

```diff
@@ -6,7 +6,7 @@ matrix:
       env:
       - BUILD_REPO_URL=https://bitbucket.org/Torr_Samaho/zandronum-stable
       - BUILD_REPO_TAG=ZA_3.0.1
-      - BUILD_IMAGE_TAGS="official-latest official-3.0"
+      - BUILD_IMAGE_TAGS="official-latest official-3.0.1"
```

Comment:
I leave out the third component of the version number on purpose because anyone using 3.0 should get hotfix releases without having to change to another tag. What problem are you trying to solve?

Reply:
Updated